### PR TITLE
Clearer error message for case in which grails server is not running, Fixes: #1187

### PR DESCRIPTION
**Pull request by febbraiod**

### CODE REVIEW

**Comment by febbraiod** [comment]
Not sure there is anything else to be done here. The submit button will not be shown if the IdentityService does not have a known user, and since this occurs before IdentityService is inited we don't have to worry about getting a second error on an attempted submit.

I tried to think of something that might be less Jargon-y than 'UI server is unavailable' but the only thing I could think of was 'Application is unavailable', but that's so general as to become meaningless and totally unhelpful.

File: core/XH.js [hunk 1]
```diff
@@ -467,7 +467,7 @@ class XHClass {
             try {
                 await this.installServicesAsync(EnvironmentService);
             } catch (e) {
-                throw `Unable to load environment info - is the server running and reachable? (${e.message})`;
+                throw `UI Server is unavailable - please contact support. (${e.message})`;
```
Comment (lbwexler) [inline core/XH.js]: Any reason we aren't using XH.exception() here?

Comment (lbwexler) [reply core/XH.js]: We could do something like XH.exception({
message: 'UI Server is unavailable - please contact support.'
detail: Some details of exception including the actual *URL*
})

Would ideally allow users to copy all details to clipboard, if not mail them  (although not sure we have easy way to do one without the other, and recall, mailing unlikely to work without backend)

Comment (febbraiod) [reply core/XH.js]: Did this. User was always, and still is, able to copy the details to the clipboard, but yes would have to email with another service. No backend to save to, and cannot set up email with only JS.

Screen Shot 2019-07-05 at 9 15 43 AM

Comment (febbraiod) [reply core/XH.js]: The latest:

Screen Shot 2019-07-05 at 10 01 00 AM

**Pull request merged**